rocksdb_admin/rocksdb_admin.thrift

```diff
@@ -170,6 +171,16 @@ struct StopMessageIngestionResponse {
   # for future use
 }
 
+enum KafkaOperationCode {
+  PUT = 1,
+  DELETE = 2
+}
+
+struct KafkaMessagePayload {
+  1: required KafkaOperationCode op_code,
+  2: optional binary value
```

Comment:
To optimize it, we may use std::unique_ptr<IOBuf> instead of binary.

There are some examples in cosmos that demonstrate how to do it.